libs/ballot-interpreter/src/hmpb-rust/interpret.rs

```diff
@@ -699,10 +692,21 @@ mod test {
 
     #[test]
     fn test_inferred_missing_metadata_from_one_side() {
-        let (side_a_image, side_b_image, options) = load_ballot_card_fixture(
-            "alameda-test",
-            ("scan-skewed-side-a.jpeg", "scan-skewed-side-b.jpeg"),
+        let (mut side_a_image, side_b_image, options) = load_ballot_card_fixture(
+            "2023-05-09-nh-moultonborough",
+            ("write-ins-front.jpeg", "write-ins-back.jpeg"),
         );
+
+        let detected = qr_code::detect(&side_a_image, &ImageDebugWriter::disabled()).unwrap();
+        let qr_code_bounds = detected.bounds();
+
+        // white out the QR code on side A
```

Comment:
Rather than relying on knowing that the image has a QR code that isn't found, I deliberately create an image without the QR code.